package.json

```diff
@@ -90,7 +90,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "yarn run prettier:check && yarn run lint"
+      "pre-push": "yarn run prettier:check && yarn run lint && yarn run validate-queries"
```

Comment:
This is debatable. Do we want to run this on push? I think so.

Reply:
I mean, we don't currently. I'm okay with adding it since I always push with `--no-verify` anyway.

Reply:
Let's ask the team

Reply:
@dpatil-magento can we add this script to the CI checks?

Reply:
https://jira.corp.magento.com/browse/PWA-341 was created -- team says NO to prepush :D